lib/Constants.sol

```diff
@@ -33,6 +33,18 @@ int constant NUM_LEVEL0 = NUM_LEVEL1 * LEVEL1_SIZE;
 int constant NUM_LEAFS = NUM_LEVEL0 * LEVEL0_SIZE;
 int constant NUM_TICKS = NUM_LEAFS * LEAF_SIZE;
 
+// FIXME: Should these be here or placed somewhere else? Besides being useful in tests, they serve as documentation for the datastructure
```

Comment:
@adhusson Do you have an opinion on where these constants should be?

Reply:
If they're not used by production code I don't think they should be there. Documentation can help fill in the gaps.

Reply:
Addressed in 9f6a98e5c4f29483f8edb7959876ee640497e7ab